packages/@o3r/core/schematics/library/index.ts

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
why do you have to disable it for the whole file ?

Reply:
No idea why it is here, it is even not needed :S